tox.ini

```diff
@@ -26,9 +31,10 @@ deps =
     django18: django>=1.8,<1.9
     django19: django>=1.9,<1.10
     django110: django>=1.10,<1.11
+    django111: django>=1.11,<2.0
     -r{toxinidir}/requirements/test.txt
 commands =
-    py.test {posargs}
+    py.test -Wd {posargs}
```

Comment:
Default warning output is deprecated so we are recommended to be explicit in order to help us prepare for Django 2.0.